composer/utils/object_store/uc_object_store.py

```diff
@@ -55,9 +55,13 @@ def __init__(self, path: str) -> None:
         except ImportError as e:
             raise MissingConditionalImportError('databricks', conda_package='databricks-sdk>=0.8.0,<1.0') from e
 
-        if not 'DATABRICKS_HOST' in os.environ or not 'DATABRICKS_TOKEN' in os.environ:
-            raise ValueError('Environment variables `DATABRICKS_HOST` and `DATABRICKS_TOKEN` '
-                             'must be set to use Databricks Unity Catalog Volumes')
+        try:
+            self.client = WorkspaceClient()
```

Comment:
so this automatically looks for those env vars or the config file?

Reply:
Yes, since even in the previous case we weren't explicitly passing the data from the env vars, just checking if they exist. Which meant that other allow forms of authentication (eg: with config files) didn't work with this code even though the underlying sdk accepts it.